js/sandbox.js

```diff
@@ -33,7 +33,7 @@ var Sandbox =  {
                 }         
             },
             listen : function (evts) {
-                if (core.is_obj(evts)) {
+                if (core.is_obj(evts) || core.is_arr(evts)) {
```

Comment:
Isn't sandbox a third party lib? I'm not crazy about modifying it incase we are breaking some license. Also, its gonna be hell upgrading the lib in the future. We just just enforce events are in an object